src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/WebservicePage/webservice.html.twig

```diff
@@ -28,58 +28,71 @@
 {% set webserviceConfigurationForm = form.webservice_configuration %}
 
 {% block content %}
-  {{ form_start(form, {'attr' : {'class': 'form', 'id': 'configuration_form'} }) }}
   <div class="row justify-content-center">
+    {% block webservice_grid %}
+      <div class="col-md-12">
+        <div class="col">
+          {{ include('@PrestaShop/Admin/Common/Grid/grid_panel.html.twig', {'grid': grid }) }}
+        </div>
+      </div>
+    {% endblock %}
+
     {% block webservice_form_configuration %}
-      <div class="col-xl-10">
-        <div class="card">
-          <h3 class="card-header">
-            <i class="material-icons">settings</i> {{ 'Settings'|trans({}, 'Admin.Global') }}
-          </h3>
+      {{ form_start(form, {'attr' : {'class': 'form', 'id': 'configuration_form'} }) }}
+        <div class="col-xl-10">
+          <div class="card">
+            <h3 class="card-header">
+              <i class="material-icons">settings</i> {{ 'Settings'|trans({}, 'Admin.Global') }}
+            </h3>
 
-          <div class="card-block row">
-            <div class="card-text">
-              <div class="form-group row">
-                <label class="form-control-label">{{ 'Enable PrestaShop\'s webservice'|trans }}</label>
+            <div class="card-block row">
+              <div class="card-text">
+                <div class="form-group row">
+                  <label class="form-control-label">{{ 'Enable PrestaShop\'s webservice'|trans }}</label>
 
-                <div class="col-sm">
-                    {{ form_errors(webserviceConfigurationForm.enable_webservice) }}
-                    {{ form_widget(webserviceConfigurationForm.enable_webservice) }}
+                  <div class="col-sm">
+                      {{ form_errors(webserviceConfigurationForm.enable_webservice) }}
+                      {{ form_widget(webserviceConfigurationForm.enable_webservice) }}
 
-                  <small class="form-text">
-                      {{ 'Before activating the webservice, you must be sure to: '|trans({}, 'Admin.Advparameters.Help') }}
-                      <br/>
-                      1. {{ 'Check that URL rewriting is available on this server.'|trans({}, 'Admin.Advparameters.Help') }}
-                      <br/>
-                      2. {{ 'Check that the five methods GET, POST, PUT, DELETE and HEAD are supported by this server.'|trans({}, 'Admin.Advparameters.Help') }}
-                    </small>
-                  </div>
-              </div>
+                    <small class="form-text">
+                        {{ 'Before activating the webservice, you must be sure to: '|trans({}, 'Admin.Advparameters.Help') }}
```

Comment:
Watch the blank space at the end of the wording. ;-)

Reply:
yes, would be great to do. Unfortunately I'll lose existing translation in this line :( 
Thanks for review, for new translations I'll have this in mind :+1:

Reply:
Perfect, thanks !